pkg/knative/ingress.go

```diff
@@ -29,18 +29,8 @@ func MarkIngressReady(ingress *networkingv1alpha1.Ingress) {
 	internalDomain := domainForServiceName(config.InternalServiceName)
 	externalDomain := domainForServiceName(config.ExternalServiceName)
 
-	var domain string
-
-	if ingress.Spec.Visibility == networkingv1alpha1.IngressVisibilityClusterLocal {
-		domain = internalDomain
-	} else {
-		domain = externalDomain
-	}
-
 	ingress.Status.MarkLoadBalancerReady(
-		[]networkingv1alpha1.LoadBalancerIngressStatus{{
-			DomainInternal: domain,
-		}},
+		nil,
```

Comment:
Why did this change?

Reply:
Public or Private LB was assigned to `ingress.status.LoadBalancer` based on `ingress.Spec.Visibility` but it is no longer necessary as `ingress.Spec.Visibility`  is deprecated.
I opened the issue against https://github.com/knative/networking/issues/136 to make it deprecate.